src/v/pandaproxy/schema_registry/service.h

```diff
@@ -39,13 +40,15 @@ class service {
     configuration& config();
     kafka::client::configuration& client_config();
     ss::sharded<kafka::client::client>& client() { return _client; }
+    store& schema_store() { return _store; }
```

Comment:
no `const store& schema_store() const` version?

Reply:
YAGNI, for now.